File: BOSSCODER_Sessions/Session_7_String_31_May/ReverseWordsInString.js

var solve = function (s) {
 let revStr = "";
    //  let s = str.split(" ");
      for(let i=s.length-1;i>=0;i--){
          revStr = revStr+s[i];
      }


      return revStr;
    ;}

    let str = "bosscoder quiz practice code";
    console.log(solve(str));

File: Bosscoder_Module_1/BegineerModule/Session_7_String_31_May/ReverseWordsInString.java
package Bosscoder_Module_1.BegineerModule.Session_7_String_31_May;

public class ReverseWordsInString {

  static public String reverseString(String str){
      String revStr = "";
      String[] s = str.split(" ");
      for(int i=s.length-1;i>=0;i--){
          revStr = revStr+s[i]+" ";
      }


      return revStr;
  }
  public static void main(String[] args) {
      String s = "bosscoder quiz practice code";
    System.out.println(reverseString(s));


  }
  
}
